packages/perseus/src/widgets/interactive-graphs/mafs-styles.css

```diff
@@ -59,6 +59,20 @@
     stroke-width: 6px;
 }
 
+.MafsView .movable-point {
+    cursor: grab;
+    touch-action: none;
+    outline: none;
+}
+
+.MafsView .movable-point.dragging {
+    cursor: grabbing;
+}
+
+.MafsView .movable-point.dragging .movable-point-halo {
+    opacity: 0;
+}
```

Comment:
My hope is that we will be adopting CSS Modules to replace Aphrodite as part of the Next Web Platform project. If we do, that should prevent all class name conflicts. In the meantime, I've modified the class name to what you suggested: `movable-point--dragging`.

Reply:
One important thing to clarify about the next CSS approach. I'm going to be experimenting with CSS modules along with another approach that is similar to Aphrodite (Panda CSS). We haven't decided which approach we are going to take yet, but this decision will be driven by different factors in a more in depth experiment that, hopefully will help us get more answers to have a more informed decision.

All that to say that we might go one or the other way (hard to say right now), but at this point, it would be preferred to continue supporting the Aphrodite approach as much as we can. The fragmentation that we have had in the past with different CSS approaches has bitten us a lot (LESS, plain css, CSS-in-JS), and having an more centralized and uniform approach helps reducing that fragmentation (at least for the time being). It will also help us migrating from one approach to another when we start working on the next platform (e.g. writing codemods to migrate one approach will have less friction than having to create codemods for different approaches).